js/slider.js

```diff
@@ -1,21 +1,22 @@
 
 const sliderElement = document.querySelector('.ad-form__slider');
 const valueElement = document.querySelector('#price');
+const maxValue = 100000;
 
 noUiSlider.create(sliderElement, {
   start: 0,
   step: 1000,
   range: {
     'min': 0,
-    'max': 100000,
+    'max': maxValue,
```

Comment:
В объектах ключи и так строки поэтому допом не оборачиваем
```suggestion
    max: MAX_VALUE,
```